tests/installation/addon_products_sle.pm

```diff
@@ -68,30 +68,28 @@ sub run() {
             send_key 'alt-u';                                                   # select URL field
             type_string get_var("ADDONURL_$uc_addon");                          # repo URL
             send_key $cmd{next};
-            if (get_var("ADDONS")) {
-                if (get_var("BETA_$uc_addon")) {
-                    assert_screen "addon-betawarning-$addon";
+            my @tags = ("addon-products", "addon-betawarning-$addon", "addon-license-$addon");
+            check_screen(\@tags, 90);
```

Comment:
A timeout of 90s in general is fine, but only on an assert_screen, not check_screen which can waste 90s in case when everything is still ok but the screen just does not show up. Can you replace the check_screen by an assert_screen and include in the @tags every potential tag that is expected?